src/migrations/Version20241105184029.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20241105184029 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```

Comment:
You should ideally "name" your migrations so you know what they contain when you return to the code later.